setup.py

```diff
@@ -0,0 +1,29 @@
+import setuptools
+
+with open("README.txt", "r", encoding="utf-8") as fhand:
+    long_description = fhand.read()
+
+setuptools.setup(
+    name="Atoms Comparsion Tool CLI",
+    version="0.0.1",
+    author="Atoms of Confusion team",
+    author_email="",
+    description=("The command line interface version of the orginal tool"),
```

Comment:
Let's combine this and the previous implementation and name this `Atoms Comparison Tool`, not the CLI